NuGet.Config

```diff
@@ -6,6 +6,8 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <!-- TODO: Mirror WixToolset.Dtf.WindowsInstaller.Package and dependencies -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This should be resolved before this PR is merged.

Reply:
Here are the instructions for [mirroring a package](https://github.com/dotnet/arcade/blob/main/Documentation/MirroringPackages.md). Also be sure to revert this file.